app/views/pages/security.html.erb

```diff
@@ -0,0 +1,211 @@
+<% @title = "Security" %>
+
+<div class="l-colspan t-body">
+  <p>RubyGems.org Security</p>
+
+  <h2>Reporting a Rubygems or RubyGems.org security bug</h2>
+
+  <p>
+    For Rails, see <a href="http://rubyonrails.org/security/">Rails Security</a>
+  </p>
+
+  <h3>Getting a CVE</h3>
+
+  <p>
+    See the <a href="http://guides.rubygems.org/security/#reporting-security-vulnerabilities">
```

Comment:
That page is really outdated... I'll see about updating https://github.com/rubygems/guides/blob/gh-pages/security.md sometime soon.

Would think ruby-security-ann@, rubysec-announce@, and oss-security@ would be better places to mention issues. Also, always a good idea to get an entry added to https://github.com/rubysec/ruby-advisory-db/ so other people will know (and those who use the bundler-audit gem will get a notification).